lighthouse-service/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/cloudevents/sdk-go v0.10.0
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-openapi/swag v0.19.5
```

Comment:
I'm wondering why this dependency was added. Any idea?

Reply:
That slipped through by copy&pasting a code snippet from another service. Has just been removed